chutney/ui/src/app/shared/components/execute-modal/scenario-execute-modal.component.ts

```diff
@@ -68,6 +134,43 @@ export class ScenarioExecuteModalComponent implements OnInit {
         }
     }
 
+    saveDataset(datasetToSave): Observable<Dataset> {
+        return this.datasetService.save(datasetToSave).pipe(
+            catchError(error => {
+                this.errorMessage = error.message
+                console.error('Error while saving dataset : ', error);
+                return of(null);
+            })
+        )
+    }
+
+    validateDatasetCreation(dataset: Dataset) {
+        return this.editionDatasetValues && this.datasetForm.get("saveDatasetName").valid &&
```

Comment:
I prefer to validate only  with reactive forms and not to use both form and value validation